plugins/ascii/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "ascii",
+    "private": true,
+    "version": "0.0.0",
+    "type": "module",
+    "main": "src/api/index.ts",
+    "scripts": {
+        "dev": "vite",
+        "build": "vite build --base=${PREFIX_BASE_PATH:+/$npm_package_name}/",
+        "format": "prettier . --write",
+        "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+        "preview": "vite preview",
+        "typecheck": "tsc --noEmit"
+    },
+    "dependencies": {
+        "@radix-ui/react-slider": "^1.2.0",
+        "@radix-ui/themes": "^3.1.3",
+        "clsx": "^2.1.1",
+        "colorjs.io": "^0.5.2",
+        "framer-plugin": "^0.3.2",
+        "ogl": "^1.0.8",
+        "react": "^18",
+        "react-dom": "^18",
+        "react-dropzone": "^14.2.3",
+        "vite-plugin-mkcert": "^1.17.5"
+    },
+    "devDependencies": {
+        "@swc/cli": "^0.4.0",
+        "@swc/core": "^1.7.14",
+        "@types/react": "^18",
+        "@types/react-dom": "^18",
+        "@typescript-eslint/eslint-plugin": "^6",
+        "@typescript-eslint/parser": "^6",
+        "@vitejs/plugin-react-swc": "^3.6.0",
+        "eslint": "^8",
+        "eslint-plugin-react-hooks": "^4",
+        "eslint-plugin-react-refresh": "^0",
+        "prettier": "^3.2.5",
+        "prettier-plugin-organize-imports": "^3.2.4",
+        "typescript": "^5.4.5",
+        "vite": "^5.2.9",
+        "vite-plugin-framer": "^0.0.9"
+    },
+    "optionalDependencies": {
+        "@rollup/rollup-linux-x64-gnu": "4.9.5"
```

Comment:
Is this being used anywhere?

Reply:
it was needed to build the project into vercel